cmd/aries-agentd/startcmd/start.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package startcmd
+
+import (
+	"github.com/gorilla/mux"
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
+	"github.com/hyperledger/aries-framework-go/pkg/framework/aries"
+	"github.com/hyperledger/aries-framework-go/pkg/framework/aries/defaults"
+	"github.com/hyperledger/aries-framework-go/pkg/restapi"
+	"github.com/spf13/cobra"
+	"net/http"
+)
+
+const AgentHostFlagName = "api-host"
+const AgentHostFlagShorthand = "a"
+const AgentHostFlagUsage = "Host Name:Port"
+
+const AgentInboundHostFlagName = "inbound-host"
+const AgentInboundHostFlagShorthand = "i"
+const AgentInboundHostFlagUsage = "Inbound Host Name:Port"
+
+const AgentDBPathFlagName = "db-path"
+const AgentDBPathFlagShorthand = "d"
+const AgentDBPathFlagUsage = "Path to database"
+
+func Cmd() *cobra.Command {
+	startCmd := &cobra.Command{
+		Use:   "start",
+		Short: "Start an agent",
+		Long:  `Start an Aries agent controller`,
+
+		Run: func(cmd *cobra.Command, args []string) {
+			host, _ := cmd.Flags().GetString(AgentHostFlagName)
```

Comment:
Need to handle error here. The startAgent will fail with "Unable to start aries agentd, host not provided" error when host is passed as empty, without giving the actual error message to the user.

Same for other cases.